### PR TITLE
[ci] Enable Vivado access on FPGA runner

### PR DESCRIPTION
**Pull request by milesdai**
The FPGA tests need to splice OTP images into bitstreams on the fly. This PR gives the FPGA runners access to the `updatemem` command.

### CODE REVIEW

**Review by alphan — APPROVED** [review]

**Review by dmcardle — APPROVED** [review]
LGTM! (Don't forget to resolve the TODO comment though :))

File: azure-pipelines.yml
Comment (milesdai) [inline azure-pipelines.yml:535]: TODO: remove before merging

Comment (alphan) [reply azure-pipelines.yml:535]: I saw the expected output in CI logs. I'll probably update and merge this PR to unblock splicing in CI. Thanks @milesdai

**Comment by alphan** [comment]
image

image

**Comment by alphan** [comment]
image

**Comment by alphan** [comment]
Dropped the last commit, verified the commit hashes, merged since CI already passed. Please see the audit trail above.

**Pull request merged**
